packages/tenant-process/src/routers/TenantRouter.ts

```diff
@@ -340,14 +341,36 @@ const tenantsRouter = (
       }
     )
     .delete(
-      "/tenants/:tenantId/attributes/verified/:attributeId",
+      "/tenants/attributes/declared/:attributeId",
```

Comment:
Why this unrelated change?

Reply:
When moving the routes to the various routers, something must have happened with the various merges, in any case everything is in order within the specific PRs

Reply:
Fixed